pytorch_lightning/trainer/trainer.py

```diff
@@ -1010,6 +1011,7 @@ def _run_train(self) -> None:
                 self.training_type_plugin.reconciliate_processes(traceback.format_exc())
             # give accelerators a chance to finish
             self.accelerator.on_train_end()
+            self._on_expection()
```

Comment:
Should we also do this for `KeyboardInterrupt`?

Reply:
Great question, I would prefer not until a user requests it.